peachjam/static/stylesheets/components/_document-content.scss

```diff
@@ -298,3 +298,15 @@
 .document-metadata-list {
   margin-bottom: 0px;
 }
+
+.document-metadata-list__row {
```

Comment:
`div` can't be a child of `dl`. Just move these styles into `.document-metadata-list`?